client/utils/challenge-types.ts

```diff
@@ -1,24 +1,19 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
 const html = 0;
 const js = 1;
-const backend = 2;
+export const backend = 2;
```

Comment:
I do not understand why these changes are here.

Reply:
They were being exported somewhere else, I changed it from a commonjs to an ESM export.

Reply:
This is being exported on line 19, already.

Reply:
I can't re-export a const so instead of doing it around line 19 I did it where it was initialized.